### ISSUE TITLE
--max-transfer  and chunker remote transfers double counting

### ISSUE THREAD

**Issue by kapitainsky** [rclone/rclone#7183]
As stats now count both real transfers and server side moves in case of chunker remote everything is double counted as every file is initially saved to tmp and then moved to final version. As a result `--max-transfer` flag does not work as expected (it stops transfers after 50% of set value)

Most likely the same behaviour can be expected for recently added feature `partial files`: transfers to a temporary file for local,ftp,sftp

discussed on the forum:

https://forum.rclone.org/t/problems-with-progress-transferred-bytes-and-speed/39773/3

**Comment by ncw** [comment]
This is a strong argument for splitting the server side stats from the normal stats.

**Comment by kapitainsky** [comment]
Given various people' requirements I would suggest two new flags:

```
--stats-ignore-server-side-move
--stats-ignore-server-side-copy
```

This would give users flexibility what to include or not.

**Comment by ncw** [comment]
I was thinking of adding a new server side moves &or copies lines in the stats block. This wouldn't appear unless you did server side moves or copies.

**Comment by kapitainsky** [comment]
yes this is even better

**Comment by ncw** [comment]
I had a go with this - please give it a test!

[v1.64.0-beta.7191.b3c480be2.fix-7183-server-side-accounting](https://beta.rclone.org/branch/fix-7183-server-side-accounting/v1.64.0-beta.7191.b3c480be2.fix-7183-server-side-accounting/) on branch [fix-7183-server-side-accounting](https://github.com/rclone/rclone/tree/fix-7183-server-side-accounting) (uploaded in 15-30 mins)

It should show an extra line in the stats like this

```
Transferred:              0 B / 20.001 MiB, 0%, 0 B/s, ETA -
Checks:                22 / 37, 59%
Transferred:           10 / 69, 14%
Server Side:           10, 46.003 MiB (copies), 0, 0 B (moves)      <<<<<<<<<<<<<<<<<<
Elapsed time:        10.0s
```

Not sure I like the `,` separator between the number `10` and the bytes `64.003 MiB` above. Maybe an `@` sign? Or something else?

**Comment by kapitainsky** [comment]
Have tried on my union/crypt/chunker remotes chain:

```
2023/07/30 19:23:46 ERROR : Attempt 2/3 succeeded
2023/07/30 19:23:46 INFO  :
Transferred:   	  532.320 MiB / 532.320 MiB, 100%, 429.098 KiB/s, ETA 0s
Checks:               412 / 412, 100%
Renamed:              204
Transferred:          204 / 204, 100%
Server Side:            0, 0 B (copies), 204, 532.182 MiB (moves)
Elapsed time:     13m30.5s
```
Looks good. I will keep it using from now on to validate in real life:)

**Comment by kapitainsky** [comment]
> Server Side:            0, 0 B (copies), 204, 532.182 MiB (moves)
> Not sure I like the `,` separator between the number `10` and the bytes `64.003 MiB` above. Maybe an `@` sign? Or something else?

I see what you mean...

what about:

`Server Side:            0 copies - 0 B, 204 moves - 532.182 MiB`

hopefully some more people on the forum will test and provide some feedback.

**Comment by darthShadow** [comment]
Added to the 1.64 milestone since we already have a tentative build.

**Comment by ncw** [comment]
After fiddling around with various formats

```
Server Side:            0, 0 B (copies), 204, 532.182 MiB (moves)
Server Side:            0 copies: 0 B, 204 moves: 532.182 MiB
Server Side:            0 (copies 0 B), 204 (moves 532.182 MiB)
Server Side:            0 copies 0 B, 204 moves 532.182 MiB
```

I decided to go for two lines. The Moves/Copies lines will only appear if non zero which I think will make things easier to understand. The `@` sign giving a definite visual delimiter with approximately the correct semantics!

```
Server Side Copies:     17 @ 532.120 MiB
Server Side Moves:       9 @ 32.320 MiB
```

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.64
